test/puppetlabs/puppetdb/command_broadcast_test.clj

```diff
@@ -156,7 +156,7 @@
 (deftest cmd-broadcast-with-all-pgs-down
   (with-open [pg1 (int/setup-postgres)
               pg2 (int/setup-postgres)
-              pdb1 (int/run-puppetdb pg1 {:database {"gc-interval" 0}
```

Comment:
Looks like we only need to pass gc-interval in as a string in tests if it's a float.